src/HModel.cpp

```diff
@@ -59,14 +66,27 @@ int HModel::load_fromMPS(const char *filename)
   modelName = filename;
 
   //setup_loadMPS(filename);
-
-  int mxNumRow = -1;
-  int mxNumCol = -1;
-  int RtCd = readMPS(filename, mxNumRow, mxNumCol,
+  // Here differentiate between parsers! 
+#ifdef Boost_FOUND
+  int RtCd = readMPS(filename, 
                      numRow, numCol, objSense, objOffset,
                      Astart, Aindex, Avalue,
-                     colCost, colLower, colUpper, rowLower, rowUpper,
-                     integerColumn);
+                     colCost, colLower, colUpper, rowLower, rowUpper);
+#else
+  int RtCd = readMPS(filename, -1, -1, 
+                     numRow, numCol, objSense, objOffset,
+                     Astart, Aindex, Avalue,
+                     colCost, colLower, colUpper, rowLower, rowUpper, integerColumn);
```

Comment:
This distinction should not be required. Maybe use a default argument in the header?